content/04.results.md

```diff
@@ -46,3 +46,14 @@ These results suggest that selecting target genes for mutation prediction based
 **B.** Overall distribution of performance across 3 gene sets. Each data point represents mean cross-validated AUPR difference, compared with a baseline model trained on shuffled mutation presence/absence labels, for one gene in the given gene set; notches show bootstrapped 95% confidence intervals. "random" = 107 random genes, "most mutated" = 107 most mutated genes, "Vogelstein et al." = 107 cancer related genes from Vogelstein et al. 2013 gene set.
 **C, D, E.** Volcano-like plots showing mutation presence/absence predictive performance for each gene in each of the 3 gene sets. The _x_-axis shows the difference in mean AUPR compared with a baseline model trained on shuffled labels, and the _y_-axis shows _p_-values for an unpaired _t_-test comparing cross-validated AUPR values.
 ](images/figure_2.png){#fig:expression_gene_sets width="90%"}
+
+### Comparing six different readouts favors expression and DNA methylation
+
+When we constructed a heatmap depicting predictive performance for each gene across data types, we found that very few genes tended to be well-predicted exclusively by one or two data types (Figure {@fig:heatmap}).
+That is, if a classifier for a given gene outperforms the permuted baseline for one data type, it is likely that it also outperforms the permuted baseline for other data types (corresponding to continuous vertical lines of boxes in the heatmap), supporting the observation that choosing the "correct" data modality is unimportant for driver genes with strong functional signatures.
```

Comment:
This _could_ be quantified - and maybe should for the skeptical reviewer?

Reply:
Makes sense. You mean something like "X/Y genes are well-predicted using only 1 or 2 data types (where X will be small and Y will be larger)"? Or some kind of statistical test?